test/parallel/test-stream-readable-event.js

```diff
@@ -113,3 +113,11 @@ const Readable = require('stream').Readable;
     assert.deepStrictEqual(result, expected);
   }));
 }
+
+{
+  const r = new Readable();
+
+  r.removeAllListeners();
```

Comment:
What if you remove this line right here? Do the tests still pass? They probably should because we never added any listeners to begin with. Could you try adding a few listeners, assert that `r.eventNames()` is non-zero, call this line and then assert it's zero please?

Reply:
OK，I'll fix it

Reply:
I've changed the tests a little bit. Any suggestions?